frontend/lib/utils/transaction-utils.ts

```
@@ -36,21 +36,96 @@ export function transformTransaction(
+  transactionData: TransactionData
+) {
+
+  switch(transactionData.type) {
+    case TransactionType.TOKEN_SEND: {
+      return {
+        type: transactionData.type,
+        token: contractAddressToHex(transactionData.token),
+        amount: transactionData.amount,
+        recipient: contractAddressToHex(transactionData.recipient),
+      }
+      break;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)